decisions/log/0034-common-urn-structure-productIDs.md

```diff
@@ -0,0 +1,214 @@
+
+Table: PACT Set of recommended `$entity-idtype$`
+
+|entity-idtype |Description      |Notes             |
+|--------------|-----------------|------------------|
+|`product-id`    | Specifies a product id from a third-party organization, standard, etc. |Use this entity-idtype when no other more specific one is applicable.|
+|`category-id`   | Indicates a category as grouping of several substances or products|Issuer for the identifier, i.e. CAS.org|
+|`buyer-id`      |Specifies a product id created by the buyer, aka “data recipient”|This is the equivalent of "buyer-assigned" as referenced in Tech Specs V2.|
+|`supplier-id`   | Specifies a product id created by the supplier, aka “data owner”|This is the equivalent of "vendor-assigned" as referenced in Tech Specs V2|
+
+#### Considerations on using 'pact' namespace
+
+As a community, we will agree on an appropriate namespace, which we could then apply to register with IANA. The basic requirements of a URN namespace are:
+ - unique and descriptive
+ - not be easily confused with existing namespaces
+ - short and memorable, ideally reflecting the name or purpose of the organization or entity
+
+Typically, namespace identifiers either refer to the organization which defines the given namespace (such as “ieee”, “iso”), or refers to the identifier itself (“epc”, "uuid”, “isbn”, etc.)
+
+After having gathered feedback on two options:
+- **pfi**
+  pfi could stand for “PACT Footprint Identification” or “Product Footprint Identification” (depending on community preference). The namespace would include all identifiers which facilitate the exchange of product (sustainability) footprint information
+- **pact**
+  PACT stands for “Partnership for Carbon Transparency” and would be a namespace to indicate all identifiers relevant for the industry-agnostic exchange of product carbon footprint information.
+
+After consultation with the PACT community, no strong opinions formed in favor or against the above two options, although a general preference formed in favor of **pact**. We recommend therefore to proceed with **pact** as the recommended namespace option. This is our recommendation given **pact** is a term our community is already familiar with, whereas **pfi** introduces a new abbreviation which could cause more confusion.
+
+
+### Part 2: Examples
+
+The below examples are proposed for inclusion in the PACT Tech Specs
+Product ID Examples
+
+#### Ex 1: Custom Product Ids (Product Codes)
+This example would replace https://wbcsd.github.io/data-exchange-protocol/v2/#dt-productid-custom
+
+    urn:pact:buyerfqdn.com:buyer-id:$CUSTOM-PRODUCT-CODE$
```

Comment:
Above, the specification reads:

> The fully qualified domain consists of top-level domain, domain, and sub-domain. Ideally the fully qualified domain points to the product specification. 

Taking this in order, should this not be:

```
urn:pact:com.buyerfqdn:buyer-id:$CUSTOM-PRODUCT-CODE$
```

i.e. domains expressed in 'backwards' notation (like in Java packages)?

Reply:
You're totally right. I've removed the confusing domain breakdown, now simply referrin to the domain name RFC for all details on domain, and included an example:  `catalog.mycompany.com`